### PR TITLE
[Inventec] Add D7264 platform support

### PR DESCRIPTION
**Pull request by loshihyu**
Add Inventec D7264 platform support.

[device/inventec/x86_64-inventec_d7264q28b-r0]: New D7264 device folder and files.
[platform/broadcom]: Added D7264 in the build.
[platform/broadcom/sonic-platform-modules-inventec/d7264q28b]: New D7264 platform folder and files.
[platform/broadcom/sonic-platform-modules-inventec/debian/]: Added D7264 platform support.

Signed-off-by: Lo.Wilson 羅仕佑 IHS lo.wilson@inventec.com

How to verify it
Unit test. Inventec integration test team has done sanity test and run system test plan. Test results are good.

### CODE REVIEW

File: platform/broadcom/sonic-platform-modules-inventec/d7264q28b/modules/inv_eeprom.c [hunk 1]
```diff
@@ -0,0 +1,181 @@
+/*
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-inventec/d7264q28b/modules/inv_eeprom.c]: why not use standard at24 eeprom driver?

File: platform/broadcom/sonic-platform-modules-inventec/d7264q28b/modules/inv_swps.h [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+#define SWP_DEV_PORT          "port"
+#define SWP_DEV_MODCTL        "module"
+#define SWP_RESET_PWD         "inventec"
+#define SWP_POLLING_PERIOD    (300)  /* msec */
+#define SWP_POLLING_ENABLE    (1)
+#define SWP_AUTOCONFIG_ENABLE (1)
+
+/* Module information */
+#define SWP_AUTHOR            "Neil <liao.neil@inventec.com>"
+#define SWP_DESC              "Inventec port and transceiver driver"
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-inventec/d7264q28b/modules/inv_swps.h]: transceiver functionality is provided by sfputil, sff8436 driver, why have your own transceiver driver?

File: platform/broadcom/sonic-platform-modules-inventec/d7264q28b/modules/inv_eeprom.c (hunk 1 above)
Comment (kuomarcel) [reply platform/broadcom/sonic-platform-modules-inventec/d7264q28b/modules/inv_eeprom.c]: Guohan
Our hardware implementation cannot use at24 eeprom driver. So we have to use a different eeprom driver. We intend to use this as our implementation for accessing eeprom. We can see if we can use standard AT24 driver and add the change later.

File: platform/broadcom/sonic-platform-modules-inventec/d7264q28b/modules/inv_swps.h (hunk 2 above)
Comment (kuomarcel) [reply platform/broadcom/sonic-platform-modules-inventec/d7264q28b/modules/inv_swps.h]: Guohan
Our implementation has more functionality such as error recovery and error handling on sfp devices. So we prefer to contribute this to use in SONiC.

**Comment by lguohan** [comment]
retest this please

**Pull request merged**